packaging/fedora/snapd.spec

```diff
@@ -476,6 +477,7 @@ install -p -m 0755 bin/snapctl %{buildroot}%{_bindir}/snapctl
 install -p -m 0755 bin/snapd %{buildroot}%{_libexecdir}/snapd
 install -p -m 0755 bin/snap-update-ns %{buildroot}%{_libexecdir}/snapd
 install -p -m 0755 bin/snap-seccomp %{buildroot}%{_libexecdir}/snapd
+install -p -m 0755 bin/snappy-app-dev %{buildroot}%{_prefix}/udev
```

Comment:
This should be `%{_prefix}/lib/udev` instead of `%{_prefix}/udev`.

Make sure the `%files` section is updated accordingly.